# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1541113.html
# Commit: 66d47e242915
# Full Hash: 66d47e24291516e241763a33048590f73a65e220
# Author: Kartikaya Gupta <kgupta@mozilla.com>
# Date: 2019-04-06 10:57:50
# Description:
#   Bug 1541113 - Avoid crashing content process with giant drawtarget. r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25931
# ==============================================================================

diff -r 9a74392d5741 -r 66d47e242915 gfx/tests/crashtests/1541113.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1541113.html	Fri Apr 05 16:26:59 2019 +0000
@@ -0,0 +1,7 @@
+<html><head>
+<meta http-equiv="content-type" content="text/html; charset=windows-1252"></head><body><svg>
+<filter id="a">
+<feTile></feTile>
+</filter>
+<polyline filter="url(#a)" points="2,0 4,2 1,32767"></polyline>
+</svg></body></html>
\ No newline at end of file